task 2

mishra-shruti15 opened the pull request:
Please ignore this request..

aayushibansal2001 committed:
Added my intoduction

Legedith committed:
add participant-Jatin

mansi35 committed:
Mansi

yitik-kawatra committed:
add participant-Yitik Kawatra

Tarster committed:
add participant-tarster

anujji1999 committed:
anujji1999

sanjitcodes committed:
add participant-sanjit

Commit:
add participanrt -sunitijain111

Chaitanya-Raj committed:
add participant-Chaitanya

lisa761 committed:
Merge pull request #22 from mansi35/mansi35-task2

Mansi.md

lisa761 committed:
Merge pull request #21 from Legedith/legedith-task2

add participant-Jatin

lisa761 committed:
Merge pull request #23 from yitik-kawatra/yitik-kawatra-task2

add participant-Yitik Kawatra

lisa761 committed:
Merge pull request #25 from anujji1999/anujji1999-task2

add participant-anujji1999

lisa761 committed:
Merge pull request #26 from sanjitcodes/sanjitcodes-task2

add participant-sanjit

lisa761 committed:
Merge pull request #27 from Tarster/tarster-task2

add participant-tarster

lisa761 committed:
Merge pull request #28 from sunitijain111/sunitijain111-task2

add participanrt -sunitijain111

Jhanvi12 committed:
add participant -jhanvi

Jhanvi12 committed:
add participant-jhanvi uday

Chaitanya-Raj committed:
add participant-Chaitanya

Chaitanya-Raj committed:
add participant-Chaitanya

simrankaur20 committed:
add participant-SimranKaur

Sjsingh101 committed:
Simranjeet Singh

tichnas committed:
add participant-Sanchit Arora

lisa761 committed:
Merge pull request #35 from tichnas/tichnas-task2

add participant-Sanchit Arora

raksha009 committed:
add participant-Raksha Jain

lisa761 committed:
Merge pull request #36 from Sjsingh101/SjSingh101-task2

add particpant-Simranjeet Singh

lisa761 committed:
Merge pull request #31 from mishra-shruti96/mishra-shruti96-task2

add participant-shruti

lisa761 committed:
Merge pull request #37 from raksha009/raksha009-task_2

add participant-Raksha Jain

lisa761 committed:
Merge pull request #38 from Jhanvi12/JhanviWorkingCopy-task2

add participant-Jhanvi

lisa761 commented:
Hi @mishra-shruti96 This is not the right way to update your master. Please follow the steps in Task-2. The commands that start with `fetch` is the correct way to update your master branch.

Chaitanya-Raj committed:
Update and rename Chaitanya.md to Chaitanya-Raj.md

Chaitanya-Raj committed:
add participant-Chaitanya

lisa761 committed:
Merge pull request #30 from Chaitanya-Raj/Chaitanya-Raj-task2

add participant-Chaitanya

simrankaur20 committed:
participant/SimranKaur

Akanksha1212 committed:
add participant-akanksha

lisa761 committed:
Merge pull request #40 from Akanksha1212/Akanksha1212-task2

add participant-akanksha

lisa761 committed:
Merge pull request #34 from simrankaur20/SimranKaur-task2

add participant-SimranKaur

saloniankita committed:
add participant-saloniankita

soumyaa1804 committed:
Merge pull request #41 from saloniankita/saloniankita-task2

add participant-saloniankita

simran212529 committed:
add participant-Simran

soumyaa1804 committed:
Merge pull request #42 from simran212530/simran212530-task2

add participant-Simran

ishikabansal04 committed:
add pariticipant-ishika

soumyaa1804 committed:
Merge pull request #43 from ishikabansal04/ikabansal04-task2

add pariticipant-ishika

aayushibansal2001 committed:
add participant-aayushibansal

Avinash-1501 committed:
add participant- Avinash

aayushibansal2001 committed:
add participant-aayushibansal2001

Avinash-1501 committed:
add participant- Avinash

lisa761 committed:
Merge pull request #45 from aayushibansal2001/aayushibansal2001-task2

Aayushibansal2001 task2

Yashasvi191 committed:
add participant-Yashasvi

lisa761 committed:
Merge pull request #44 from Avinash-1501/Avinash-1501-task2

add participant- Avinash

lisa761 committed:
Merge pull request #46 from Yashasvi191/Yashasvi191-task2

add participant-Yashasvi

Tanisha07 committed:
add participant-Tanisha

Omkaragrawal committed:
add participant - OMKAR AGRAWAL

lisa761 committed:
Merge pull request #47 from Tanisha07/Tanisha07-task2

add participant-Tanisha

lisa761 committed:
Merge pull request #48 from Omkaragrawal/omkaragrawal-task2

add participant - OMKAR AGRAWAL

Prerna-Goel committed:
add participant-Prerna

soumyaa1804 committed:
Merge pull request #49 from Prerna-Goel/Prerna-Goel-task2

add participant-Prerna

Legedith committed:
Markdown styling Jatin/Legedith

Prerna-Goel committed:
Styling done-Prerna

vkr-code committed:
add participant-Vanshika

Tarster committed:
Challenge 3 markdown complete

Legedith committed:
Update legedith.md

Commit:
styling - sunitijain111

lisa761 committed:
Merge pull request #52 from Legedith/styling-md-legedith

Markdown styling Jatin/Legedith

Prerna-Goel committed:
styling done - Prerna

Sjsingh101 committed:
SjSingh101

yitik-kawatra committed:
styling done yitik

mansi35 committed:
styling participant mansi35

raksha009 committed:
Styling - Raksha

lisa761 committed:
Merge pull request #55 from vkr-code/vkr-code-task2

add participant-Vanshika

tichnas committed:
Markdown Styling-Sanchit Arora

aayushibansal2001 committed:
markdown added by aayushi

raksha009 committed:
Update raksha009.md

soumyaa1804 committed:
Merge pull request #65 from tichnas/styling-md-tichnas

Markdown Styling-Sanchit Arora

anujji1999 committed:
Styling done

soumyaa1804 committed:
Merge pull request #62 from mansi35/styling-md-mansi35

styling participant mansi35

Prerna-Goel committed:
styling done - Prerna

soumyaa1804 committed:
Merge pull request #54 from Prerna-Goel/styling-md-Prerna

Styling done-Prerna

raksha009 committed:
Update raksha009.md

Jhanvi12 committed:
styling participant jhanvi

Avinash-1501 committed:
styling

mishra-shruti15 committed:
Markdown Task-3 Done

Avinash-1501 committed:
styling

soumyaa1804 committed:
Merge pull request #63 from raksha009/styling-md-Raksha-Jain

Styling - Raksha

niyati-n committed:
add participant-Niyati

aayushibansal2001 committed:
markdown styling by aayushibansal

sanjitcodes committed:
add Styling-sanjit

Akanksha1212 committed:
Updated Akanksha1212.md

Chaitanya-Raj committed:
Update Chaitanya-Raj.md

for #50

yitik-kawatra committed:
styling added

lisa761 committed:
Merge pull request #61 from yitik-kawatra/styling-md-yitikkawatra

styling done yitik

eternal-dreamer committed:
add paticipant-Shrishti GOYAL

lisa761 committed:
Merge pull request #74 from Chaitanya-Raj/styling-md-Chaitanya

Update Chaitanya-Raj.md

lisa761 committed:
Merge pull request #76 from eternal-dreamer/eternal-dreamer-task2

add participant-Shrishti Goyal

Akanksha1212 committed:
Update Akanksha1212.md

Jhanvi12 committed:
styling participant jhanvi-uday edited

Jhanvi12 committed:
Update jhanvi12.md

lisa761 committed:
Merge pull request #68 from aayushibansal2001/styling-md-aayushibansal

markdown added by aayushi

soumyaa1804 committed:
Merge pull request #73 from Akanksha1212/styling-md-Akanksha1212

Markdown styling Akanksha

ishikabansal04 committed:
Update README.md

Sjsingh101 committed:
Update SjSingh101.md

simran212529 committed:
Styling-simran

Avinash-1501 committed:
styling

Avinash-1501 committed:
styling

Avinash-1501 committed:
styling

Avinash-1501 committed:
styling

Tarster committed:
Challenge 3 complete

Yashasvi191 committed:
Edited Yashasvi191.md file

soumyaa1804 committed:
Merge pull request #60 from Sjsingh101/styling-md-SjSingh101

SjSingh101

lisa761 committed:
Merge pull request #70 from Avinash-1501/styling-md-Avinash

styling

Jhanvi12 committed:
Update jhanvi12.md

Finally pls , merge this.

lisa761 committed:
Merge pull request #82 from Tarster/styling-md-tarster

Styling md tarster

lisa761 committed:
Revert "add participant-Shrishti Goyal"

lisa761 committed:
Merge pull request #84 from ACM-IGDTUW/revert-76-eternal-dreamer-task2

Revert #76

soumyaa1804 committed:
Merge pull request #83 from Yashasvi191/styling-md-Yashasvi191

Edited Yashasvi191.md file

saloniankita committed:
Update saloniankita.md

Added intro for task-3

Tanisha07 committed:
add participant-Tanisha

simran212529 committed:
Styling-simran

Omkaragrawal committed:
TASK 3 omkaragrawal.md

ishikabansal04 committed:
Update ikabansal04.md

ishikabansal04 committed:
Update README.md

lisa761 committed:
Merge pull request #88 from Omkaragrawal/styling-md-omkaragrawal

TASK 3 omkaragrawal.md

lisa761 committed:
Merge pull request #79 from simran212530/styling-md-simran

Styling-simran

lisa761 committed:
Merge pull request #67 from Jhanvi12/styling-md-Jhanvi

styling participant jhanvi

sanjitcodes committed:
styling-sanjit

lisa761 committed:
Merge pull request #66 from anujji1999/styling-md-anujji1999

Styling done anujjj19999

sanjitcodes committed:
styling-sanjit

ishikabansal04 committed:
Update README.md

sanjitcodes committed:
styling-sanjit-modified

ishikabansal04 committed:
Update ikabansal04.md

ishikabansal04 committed:
Update README.md

ishikabansal04 committed:
Update ikabansal04.md

lisa761 committed:
Merge pull request #72 from sanjitcodes/styling-md-sanjit

add Styling-sanjit

ishikabansal04 committed:
Update README.md

soumyaa1804 committed:
Merge pull request #78 from ishikabansal04/styling-md-ishika

Update ikabansal04.md

soumyaa1804 committed:
Merge pull request #86 from saloniankita/patch-1

Markdown task-3

niyati-n committed:
Delete blackcrabb.md

niyati-n committed:
Create blackcrabb.md

lisa761 committed:
Merge pull request #71 from blackcrabb/blackcrabb-task2

add participant-Niyati

eternal-dreamer committed:
add participant-Shrishti Goyal

eternal-dreamer committed:
Update eternal-dreamer-task2.md

eternal-dreamer committed:
Delete participant.txt

Tanisha07 committed:
Task 3- Tanisha

Tanisha07 committed:
Task 3- Tanisha

Tanisha07 committed:
Task 3- Tanisha_Done

lisa761 committed:
Merge pull request #91 from eternal-dreamer/eternal-dreamer-task2

add participant-Shrishti Goyal

lisa761 committed:
Merge pull request #87 from Tanisha07/styling-md-Tanisha07

styling-Tanisha

priyanka2109 committed:
add participant priyanka

niyati-n committed:
styling Niyati

lisa761 committed:
Merge pull request #99 from priyanka2109/priyanka2109-task2

add participant priyanka

niyati-n committed:
styling niyati

niyati-n committed:
styling2 niyati

niyati-n committed:
final styling niyati

lisa761 committed:
Merge pull request #100 from blackcrabb/styling-md-blackcrabb

styling Niyati

priyanka2109 committed:
styling-priyanka

priyanka2109 committed:
Delete priyanka2109.md

mansi35 committed:
Changing Card Style 

In accordance with #93 
I've added the brightness increasing decreasing effect in the cards along with the previous card effect staying as it is. Thus, making the issue at hand more focused upon.

lisa761 committed:
Merge pull request #110 from priyanka2109/styling-md-priyanka21

styling-priyanka

Tarster committed:
removed broken css link queryloader.css

Tarster committed:
added two more links github and instagram

Legedith committed:
Added social media links

Tarster committed:
Removed Instagram Link and Removed a broken css resource file link

Legedith committed:
Update events.ejs

soumyaa1804 committed:
Merge pull request #120 from mansi35/patch-1

Changing Card Style

soumyaa1804 committed:
Merge pull request #124 from Legedith/pr-for-task-6-Legedith

Added social media links

Tarster committed:
Merge branch 'master' into finaltask-5

soumyaa1804 committed:
Merge pull request #128 from Tarster/finaltask-5

Added GitHub link and removed a broken CSS link

lisa761 committed:
Merge pull request #57 from sunitijain111/stylind-md-suitijain

styling - sunitijain111

lisa761 committed:
Merge pull request #69 from mishra-shruti96/styling-md-shruti

Markdown Task-3 Done

soumyaa1804 committed:
Update README.md

soumyaa1804 committed:
Update README.md
